doc/en/user/source/production/config.rst

```diff
@@ -107,6 +107,40 @@ To disable the Auto Complete on Web Admin login form:
 * Set the Java system property ``geoserver.login.autocomplete`` to off by adding ``-Dgeoserver.login.autocomplete=off`` to your container's JVM options
 * If the browser has already cached the credentials, please consider clearing the cache or form data after setting the JVM option.
 
+Disable anonymous access to the layer preview page
+--------------------------------------------------
+
+In some circumstances, you might want to provide access to the layer preview page to authenticated users only. The solution is based on
+adding a new :guilabel:`filter chain` with a rule matching the path of the layer preview page to GeoServer's :ref:`security_auth_chain`. Here are the
+steps to reproduce:
+
+* Under :guilabel:`Security` -> :guilabel:`Authentication` -> :guilabel:`Filter Chains`, add a new HTML chain
+* Set the new chain's name to ``webLayerPreview`` (or likewise)
+* As ANT pattern, enter the path of the layer preview page, which is :file:`/web/wicket/bookmarkable/org.geoserver.web.demo.MapPreviewPage`
+  (since it's an ANT pattern, the path could as well be written shorter with wildcards: :file:`/web/**/org.geoserver.web.demo.MapPreviewPage`)
```

Comment:
```suggestion
  (since it's an Ant pattern, the path could as well be written shorter with wildcards: :file:`/web/**/org.geoserver.web.demo.MapPreviewPage`)
```